README.md

```diff
@@ -1,2 +1,41 @@
 # vivo-es
 Configuration for ElasticSearch index used by VIVO
+
+# How to setup?
+
+## Build the Docker Image:
+
+```
+docker build -t vivo-elasticsearch .
+```
+
+## Run the Container:
+
+```
+docker run -p 9200:9200 -p 9300:9300 --name vivo_es vivo-elasticsearch
+```
+
+This setup will start Elasticsearch, expose it on localhost:9200, and create the vivo index with the specified mapping by loading the configuration from index-config.json.
+
+## Alternatively - Run Elasticsearch locally
+
+1. Download Elasticsearch (suited for your platform) from the [official website](https://www.elastic.co/downloads/elasticsearch).
+2. Run `bin/elasticsearch` (or `bin\elasticsearch.bat` on Windows) to start Elasticsearch [with security enabled](https://www.elastic.co/guide/en/elasticsearch/reference/current/configuring-stack-security.html).
+3. Create `vivo` index by running:
```

Comment:
I don't think that that is a good idea because the script expects that the ES is installed in `/usr/share/elasticsearch` which is only 100% true if someone builds the image using my Dockerfile.